### DISCUSSION TITLE
Investor email

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#432] [Announcements]
> In the true spirit of open source, this investor email is shared with everyone.

Hello DreamBerd investors (all 29 of you). 

First of all, thank you for your money. As of today, we've raised £181.81 and also a sweet wrapper[^1]. 

**And now, we're happy to announce what your money will be spent on.**

But first... a community update!

## Community update

DreamBerd's investment round went viral on micro-blogging network, mastodon. That's where the vast majority of fundraising & discussion happened.

As an after-thought, it was also cross-posted to [Elon Musk's X](https://www.thepinknews.com/2023/10/05/x-twitter-report-system-transphobia/) on twitter dot com. A small number of people enjoyed the 'sloppy seconds' of the event over there.

## What your money will be spent on

Your money will be donated to the [Stonewall charity](https://www.stonewall.org.uk/). This will help to prevent the DreamBerd creator from losing their human rights, allowing maintenance of the project to continue.

This has been part of the DreamBerd docs for a [long time](https://github.com/TodePond/DreamBerd/releases/tag/v0.9.9.9.9.9.9.9.9d). Even so, most influencers suspiciously [skip](https://youtu.be/tDexugp8EmM?t=21m36s) that section when doing their read-through.

In the past, we've [tried](https://x.com/TodePond/status/1410195019403759621?s=20) [several](https://x.com/TodePond/status/1542450085065261056?s=20) [times](https://x.com/TodePond/status/1342448997735215104?s=20) to [raise money](https://x.com/TodePond/status/1277194772432764928?s=20) for [similar causes](https://universeodon.com/@TodePond/110637724975609602). Unfortunately, all of these attempts fell flat. Very few people in the tech world seem interested in donating to an LGBT+ charity.

*But everyone seems keen to donate to an imaginary programming language.*

## Projected growth

Following this announcement, we expect a slight change in growth.

Some people will become more keen to invest, and will donate more money. Others will become disinterested, and will quietly slip away. A small number of people will become actively offended.

Whatever you decide to do — we fully support you. Please continue/stop promoting DreamBerd. We do/don't want your good/bad vibes in our community. We hope you have a lovely/awful day.

## A message from our founder

Stonewall is currently campaigning to get conversion therapy banned. If you don't know what conversion therapy is... It's like JavaScript type coercion, except it happens to queer people instead of strings.

Unfortunately, my government [refuses](https://www.stonewall.org.uk/our-work/campaigns/ban-conversion-therapy) to do anything about it. Our prime minister prefers to go on [anti-trans rants](https://x.com/redstreamnet/status/1709893999245582613?s=20).

Banning conversion therapy would be the absolute bare minimum. It's a form of psychological torture and **it happens to more people than you think**. For example, it's how I ended up as NaN-binary.

So...

_**🎵 [Invest in DreamBerd](https://dreamberd.computer)! have some fun! support esoteric programming! and fight conversion therapy! 🎵**_

> Invest now at [dreamberd.computer](https://dreamberd.computer). All of your money will be given to the Stonewall charity.

Thanks for reading this week's release notes. See you next week.

[^1]: sweet wrapper by verbal agreement only

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/v181.81'>Investor email</a>.</em>

**Comment by qexat** [comment]
> imaginary programming language

dreamberd isn't that complex, isn't it?

**Reply by exersalza** [reply]
something this perfect couldn't be complex

**Comment by slweeb** [comment]
> If you don't know what conversion therapy is... It's like JavaScript type coercion, except it happens to queer people instead of strings.

hardest line ever

**Reply by MaddyGuthridge** [reply]
Legitimately the best explanation

**Comment by prawnydagrate** [comment]
how do i install the installer installer

**Reply by mybearworld** [reply]
Install Create Dreamberd App, easier

**Reply by MaddyGuthridge** [reply]
@mybearworld `create-dreamberd-app` is essentially unmaintained at this point, so using it would be a terrible decision. Instead, you should use [borderline proprietary tool that pushes you towards expensive deployment system]
